yt_dlp/YoutubeDL.py

```diff
@@ -3071,7 +3071,7 @@ def process_subtitles(self, video_id, normal_subtitles, automatic_captions):
                 f = formats[-1]
                 self.report_warning(
                     'No subtitle format found matching "%s" for language %s, '
-                    'using %s' % (formats_query, lang, f['ext']))
+                    'using %s. Add the --list-subs flag to see a list of available subtitle formats' % (formats_query, lang, f['ext']))
```

Comment:
```suggestion
                    'using %s. Use --list-subs for a list of available subtitles' % (formats_query, lang, f['ext']))
```
to mirror the language used in `--format`'s hint